velox/exec/Task.cpp

```diff
@@ -2423,6 +2425,22 @@ void Task::onTaskCompletion() {
   });
 }
 
+TaskState Task::realState() const {
```

Comment:
Why not just change `state()`?

Reply:
@Yuhta 
We have

`TaskState state() const {`

It returns `state_` member variable.
The comment in the `Task::realState()` is trying to explain why we need `realState()` for now.